tests/core/pyspec/eth2spec/test/phase0/block_processing/test_process_attester_slashing.py

```diff
@@ -197,6 +197,74 @@ def test_participants_already_slashed(spec, state):
 # Some of the following tests are phase0 only: phase 1 lists participants with bitfields instead of index list.
 
 
+@with_phases([PHASE0])
+@spec_state_test
+@always_bls
+def test_att1_high_index(spec, state):
+    attester_slashing = get_valid_attester_slashing(spec, state, signed_1=True, signed_2=True)
+
+    indices = get_indexed_attestation_participants(spec, attester_slashing.attestation_1)
+    indices.append(spec.ValidatorIndex(len(state.validators)))  # off by 1
+    attester_slashing.attestation_1.attesting_indices = indices
+
+    yield from run_attester_slashing_processing(spec, state, attester_slashing, False)
+
+
+@with_phases([PHASE0])
+@spec_state_test
+@always_bls
+def test_att2_high_index(spec, state):
+    attester_slashing = get_valid_attester_slashing(spec, state, signed_1=True, signed_2=True)
+
+    indices = get_indexed_attestation_participants(spec, attester_slashing.attestation_2)
+    indices.append(spec.ValidatorIndex(len(state.validators)))  # off by 1
+    attester_slashing.attestation_2.attesting_indices = indices
+
+    yield from run_attester_slashing_processing(spec, state, attester_slashing, False)
+
+
+EMPTY_SIGNATURE = b'\xc0' + (b'\x00' * 95)
```

Comment:
To avoid confusion between `b'\xc0' + (b'\x00' * 95)` and `b'\x00` * 96, I used `Z2_SIGNATURE` in https://github.com/ethereum/eth2.0-specs/blob/7770accf96f5565ce3d9fc81f7191bae7421189f/tests/core/pyspec/eth2spec/utils/bls.py#L13
p.s. `Z2` is the point at infinity for the twisted curve over FQ2 in py_ecc

Or, it can be renamed to something like `G2_INFINITY_SIGNATURE`.

Reply:
approved pending deciding what to do here